physics/geopotential_body.hpp

```diff
@@ -246,7 +278,9 @@ Acceleration(OblateBody<Frame> const& body,
     }
     grad_ℜ = -(n + 1) * r * ℜ / r²;
 
-    return (... +
+    // The zero_ term lifts an ambiguity on the order of evaluation of the
+    // acceleration for orders 0 and 1.
+    return (zero_ + ... +
             DegreeNOrderM<size, degree, orders>::Acceleration(
                 body, r, precomputations));
```

Comment:
As discussed, the order of evaluation of subexpressions is undefined (which means that the comment in #1945 about MSVC being confused is incorrect; it is merely the author who is confused).
Get rid of the UB and of this `zero_`.